src/array_api_extra/_funcs.py

```diff
@@ -555,7 +555,7 @@ def sinc(x: Array, /, *, xp: ModuleType | None = None) -> Array:
 
 def pad(
     x: Array,
-    pad_width: int,
+    pad_width: int | tuple | list,
```

Comment:
I think CI is looking for something like

```suggestion
    pad_width: int | tuple[int, ...] | list[tuple[int, int], ...],
```